controller/user-controller.js

```diff
@@ -0,0 +1,16 @@
+const userService = require('../services/user-services');
+
+const signUp = async (req, res) => {
+  try {
+    const {email, password, firstname, lastname} =req.body;
+
+    await userService.signUp(email, password, firstname, lastname);
+
+    res.status(201).end();
+  }catch (err) {
+    res.status(err.statusCode || 400).json({message: err.message});
+  }
+};
+
+
+module.exports = { signUp };
```

Comment:
작성한 모듈을 내보내기 위한 코드를 좀 더 가독성을 높게 작성하는 방법은 다음과 같습니다.

하나의 파일 내부에 하나의 모듈만 존재할 수도 있고, 여러개의 모듈이 존재할 수도 있습니다.

경우에 따라 아래와 같이 작성해주시면 좋을 것 같습니다.

다른 부분에서도 해당 내용을 확인하여 적용해주세요!

```suggestion
// 모듈이 하나인 경우
// 중괄호 시작과 닫는 부분에 띄어쓰기를 넣어 모듈이 한눈에 모이도록 작성
module.exports = { signUp };

// 모듈이 하나 이상인 경우
module.exports = {
  signUp,
  signIn
};
```